src/Domain/Entities/TrustLevel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Codidact.Domain.Common;
+
+namespace Codidact.Domain.Entities
+{
+    /// <summary>
+    /// Levels of trust for each member
+    /// </summary>
+    public class TrustLevel : AuditableEntity
+    {
+        /// <summary>
+        /// Auto Incremented Identification number
+        /// </summary>
+        public long Id { get; set; }
+       
+        /// <summary>
+        /// Name of the trust level
+        /// </summary>
+        public string Content { get; set; }
```

Comment:
Can be `LevelName`, `Content` doesn't describe what the field is.

Reply:
Possibly `UserTrustLevel`.

Reply:
@Linnydude3347 That would imply it's the trust level assigned to a particular user. It's the name of a trust level, so the field name should say that.